ci/test-ldap

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex
```

Comment:
This file should be removed. It's now handled by `ci/test --cucumber-authenticators`